specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+          "description": "ID of the server hosting the datacenter management solution.",
+          "readOnly": true
+        },
+        "description": {
+          "type": "string",
+          "description": "Description of the machine",
+          "readOnly": true
+        },
+        "displayName": {
+          "type": "string",
+          "description": "User readable name of the machine as defined by the user in their private datacenter.",
+          "readOnly": true
+        },
+        "megabytesOfMemory": {
+          "type": "number",
+          "description": "Memory in Megabytes.",
+          "readOnly": true
+        },
+        "numberOfCores": {
+          "type": "integer",
+          "description": "Processor count.",
+          "readOnly": true
+        },
+        "operatingSystem": {
+          "type": "string",
+          "description": "Operating System of the machine.",
+          "readOnly": true
+        },
+        "groups": {
+          "type": "array",
+          "description": "List of references to the groups that the machine is member of.",
+          "readOnly": true,
+          "items": {
+            "type": "string",
+            "readOnly": true,
+            "description": "Reference to a group that this machine is member of. /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/projects/{projectName}/groups/{groupName}"
+          }
+        },
+        "createdTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was created. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was last updated. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "discoveredTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this machine was discovered by Azure Migrate agent. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "disks": {
+          "type": "object",
+          "description": "Dictionary of disks attached to the machine. Key is ID of disk. Value is a disk object",
+          "readOnly": true,
+          "properties": {
+            "key": {
+              "type": "string",
+              "description": "Unique identifier of a disk.",
+              "readOnly": true
+            },
+            "value": {
+              "description": "Disk object.",
+              "$ref": "#/definitions/Disk"
+            }
+          }
+        },
+        "networkAdapters": {
+          "type": "object",
+          "description": "Dictionary of network adapters attached to the machine. Key is ID of disk. Value is a network adapter object",
+          "readOnly": true,
+          "properties": {
```

Comment:
>properties [](start = 11, length = 10)

Same here, it is better not to define objects inline

Reply:
same as above.